hammer.sh

```diff
@@ -446,6 +447,34 @@ function install_deps_freealut()
     make install > $LOGDIR/deps/freealut/$INSTALLLOG
 }
 
+function install_deps_basedir()
+{
+    # libxdg-basedir
+    echo "  Installing libxdg-basedir..."
+    mkdir -p $LOGDIR/deps/libxdg-basedir
+    cd $DEPS_SOURCE
+
+    echo "  Downloading..."
+    curl -OL http://nevill.ch/libxdg-basedir/downloads/libxdg-basedir-latest.tar.gz
```

Comment:
If you use "latest", one never knows which version it will be. Stuff will break randomly.
Instead always specify the exact version.